CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## [14.4.0](https://github.com/commercetools/merchant-center-application-kit/compare/v14.3.5...v14.4.0) (2019-08-14)
+
+#### 🚀 Type: New Feature
+
+- `application-shell`, `permissions`
```

Comment:
Please next time pick the adequate label otherwise the changelog won't be generated automatically :)